rubocop/rubocop.gemspec

```diff
@@ -6,6 +6,10 @@ Gem::Specification.new do |spec|
   # x.y.z.t
   # Where x.y.z = rubocop version
   # t is incremental
+
+  # TODO: make as a separated gem hosted via rubygems
+  # TODO: rubocop-gem-non-related versioning
```

Comment:
Why?

Reply:
Rubocop ruleeses is our version - not a gem.
Change cop options => Change version - How?
Change gem dependency => Change version - OK.

Reply:
> Change cop options => Change version - How? (problem) (and this is the main point of this gem i think)

It is described in comment above: you should increment "t"

RTFM

Reply:
Weeeeeeell, Ok